### PR TITLE
[Expert] General Adjustments and Occultism Miner tweaks

### PR DESCRIPTION
**Pull request by MuteTiefling**
Change Advanced Pump Upgrade so it no longer conflicts with the XP Pump Upgrade
image
image

Cheaper Relays as they're used a lot. 4 cobble per was a bit much.
image

Revert 4x plastic processing in Blast Furnace. This made it difficult to automate as a vanilla hopper could no longer insert the recipe. Could only be automated by systems pushing 4 or more plastic in the same tick. This just makes the Arc Furnace recipe that much more relevant. 
image

Restores the old Stabilizer values. These were lost when switching to ConfigSwapper.
Also reduces the durability of the Occultism Miners for Expert only. This is to encourage automating repair so they aren't just free resources. 
Also splits out the loot tables for the two miners. The Foliot miner now gets a selection of world gen materials like clay, stones, end stone, ice, etc. It can no longer mine ore. 
The Djinni remains an Ore miner and has been updated with a few more ores. 

New basic lamp recipe
image

Miner Foliot brought earlier
image

### CODE REVIEW

**Comment by MuteTiefling** [comment]
Introduces a new dimensional mineshaft lamp that brings back fish instead of ores/stones.

Has a 10% chance to come back with a loot chest filled with goodies. (goodies yet to be finalized)

**Comment by MuteTiefling** [comment]
New Fisher Foliot for all modes
Expert
image

Normal
image


Catches a wide variety of fish and some special loot in loot chests.

Late Game alternative for smoldering lapis compound. It's very slow to process in the IE blast furnace.
image

**Comment by MuteTiefling** [comment]
Should be all set here.

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/dimensional_mineshaft/fish.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            weight: 100,
+            id: `${id_prefix}fish_bones`
+        }
+    ];
+
+    fishableFish.forEach((fish) => {
+        recipes.push({
+            output: fish,
+            weight: 100,
+            id: `${id_prefix}${fish.split(':')[0]}`
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/dimensional_mineshaft/fish.js:50]: I think the index is supposed to be 1 here in order to get the fish name.

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/occultism/shaped.js [hunk 2]
```diff
@@ -37,6 +37,15 @@ onEvent('recipes', (event) => {
                 C: '#forge:inlays/pewter'
             },
             id: 'occultism:crafting/sacrificial_bowl'
+        },
+        {
+            output: 'occultism:magic_lamp_empty',
+            pattern: [' A ', 'ABA', ' AA'],
+            key: {
+                A: '#forge:ingots/silver',
+                B: 'atum:horus_godshard'
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/occultism/shaped.js:46]: Isn't there a tag for specific godshards?

**Review by NillerMedDild — APPROVED** [review]

File: kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/dimensional_mineshaft/fish.js (hunk 1 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/dimensional_mineshaft/fish.js:50]: aha, yep. good catch

File: kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/occultism/shaped.js (hunk 2 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/occultism/shaped.js:46]: unfortunately, no. There's a genereal `atum:godshards` but nothing for individual shards

**Review by NillerMedDild — APPROVED** [review]

**Pull request merged**